app/backend/src/couchers/servicers/communities.py

```diff
@@ -245,3 +245,26 @@ def LeaveCommunity(self, request, context):
             session.query(ClusterSubscription).filter(ClusterSubscription.user_id == context.user_id).delete()
 
             return empty_pb2.Empty()
+    
+    def ListUserCommunities(self, request, context):
+        with session_scope() as session:
+            page_size = min(MAX_PAGINATION_LENGTH, request.page_size or MAX_PAGINATION_LENGTH)
+            next_node_id = int(request.page_token) if request.page_token else 0
+            nodes = (
+                session.query(Node)
+                .filter(Node.id >= next_node_id)
+                .order_by(Node.id)
+                .limit(page_size + 1)
+                .all()
+            )
```

Comment:
should I fetch more than just the communities. Should I also return the groups, pages etc or should these be separate APIs?